service-api/module/Application/src/Yoti/YotiService.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Yoti;
+
+use Application\Exceptions\YotiException;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use Laminas\Http\Response;
+
+/**
+ * @psalm-suppress PossiblyUnusedProperty
+ * Suppress unused $client pending implementation
+ */
+class YotiService implements YotiServiceInterface
+{
+    public function __construct(
+        public readonly Client $client
+    ) {
+    }
+
+    /**
+     * @param string $postCode
+     * @return array
```

Comment:
It would be good to expand these returns to indicate the shape of the array coming back (e.g. `@return array{fad_code: string, name: string, address: string[], postcode: string}[]`)

Same for other responses

Reply:
This would be the case in the controller when we do the formatting for our needs. The YotiService in this case would return the same format specified by Yoti which would be like

 `* @return array{
 *     branches: array{
 *         type: string,
 *         fad_code: string,
 *         name: string,
 *         address: string,
 *         postcode: string,
 *         location: array{
 *             latitude: float,
 *             longitude: float
 *         }
 *     }[]
 * }
 * `

I suppose?

Reply:
Yeah that looks good. I think it's worth including that in the annotation so we have code-level documentation and type-hinting.